src/data/index.ts

```diff
@@ -0,0 +1,49 @@
+import { configureStore, Dispatch } from '@reduxjs/toolkit';
+import { persistReducer, persistStore } from 'redux-persist';
+// import loggerMiddleware from 'redux-logger';
+import MementoStorage from './storage';
+
+import rootReducer, { actions } from './slice';
+import { Memento } from 'vscode';
+
+const buildStore = (workspaceState: Memento) => {
+	const persistConfig = {
+		key: 'root',
+		storage: new MementoStorage(workspaceState),
+		// debugger: console.log,
```

Comment:
perhaps temporarily we should use `debugger: console.debug` here?

Reply:
its better to remove this debugger at all, it spams too much